deploy/assisted-controller/assisted-controller-deployment.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: apps/v1
```

Comment:
i think it is very strange that deployment of assisted-controller is in the assisted-service repo. I think it should be in the assisted-installer repo, and we can update test-infra to use Makefile from that repo